imports/plugins/included/paystack/client/settings/paystack.js

```diff
@@ -0,0 +1,31 @@
+import { Template } from "meteor/templating";
+import { Reaction } from "/client/api";
+import { Packages } from "/lib/collections";
+import { PaystackPackageConfig } from "../../lib/collections/schema";
+
+import "./paystack.html";
+
+Template.paystackSettings.helpers({
+  PaystackPackageConfig() {
+    return PaystackPackageConfig;
+  },
+  packageData() {
+    return Packages.findOne({
+      name: "paystack-paymentmethod",
+      shopId: Reaction.getShopId()
+    });
+  }
+});
+
+AutoForm.hooks({
+  "paystack-update-form": {
+    onSuccess: function () {
+      Alerts.removeSeen();
+      return Alerts.add("Paystack settings saved.", "success");
+    },
+    onError: function (operation, error) {
+      Alerts.removeSeen();
+      return Alerts.add("Paystack settings update failed. " + error, "danger");
```

Comment:
I agree with @andela-aolaniran  since that's the style they adopted in the code base